### PR TITLE
[FE][FIX] make categories expandable 

### PR DESCRIPTION
**Pull request by MarcPIster**
### 🛠 Changes being made

We now have an backend endpoint to fetch all existing categories.
We then display it as a list (Max 10 at once), this list is expandable.

### ✨ What's the context?

We now see ALL categories that we have.

### 🧪 Test plan

How do you know the changes are safe to ship to production?

### 📸 Screenshots (optional)

If you made UI changes, what are the before an afters?

### 🏎 PR checklist

1. Read the ticket
- [ ] Read the related ticket on Jira and understand the requirements for this ticket. In case anything is unclear, ask the ticket creator or ticket executor.
2. Matching PR description
- [ ] Check that the PR description on GitHub is aligned with the ticket description on Jira.
3. Review the changes on GitHub
- [ ] is the current version of the destination branch (mostly main) merged into the PR branch?
- [ ] does it fulfill the definition of done for the ticket?
- [ ] were the tests adjusted or extended for the changed behavior?
- [ ] was the documentation adjusted or extended for changed behavior?
- [ ] does it align with the team code guidelines?
- [ ] check readability of code - is it easy to understand what is happening in the code snippets?
- [ ] proper naming
- [ ] is it logical?
- [ ] are there any performance issues?
- [ ] could it be improved / simplified?
4. Execute locally / in testing environment
- [ ] Execute the code locally and check if there are any issues. Test whether the new behavior from the ticket acts as expected. Try to think of edge cases and test them.
5. Run tests
- [ ] Run the tests locally and check that all of them succeed.
6. Linting & Formatting
- [ ] Check that the changes didn’t cause any linting or formatting warnings of errors.
7. Submit Feedback
- [ ] Approve if all of the following points fulfilled:
   - all previous steps did not reveal any issues
   - there are no open comments on the PR

- [ ] otherwise, request changes with specific comments (constructive) on what to change!

### CODE REVIEW

File: packages/backend/src/controllers/categorieController.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+export function categorizeAndSort(categories: string[]): string[] {
+  const categoryCounts: Record<string, number> = {};
+
+  categories.forEach(category => {
```
Comment (JosefineM2000) [inline packages/backend/src/controllers/categorieController.ts:4]: could you please add fuzziness here to group similar names? ChatGPT suggests a method like this:

```
const similarity = (s1, s2) => {
  const len = Math.max(s1.length, s2.length);
  let diff = 0;
  for (let i = 0; i < len; i++) {
    if (s1[i] !== s2[i]) diff++;
  }
  return diff / len;
};
```

File: packages/visitorWeb/src/components/Filter/Filter.module.scss
Comment (JosefineM2000) [inline packages/visitorWeb/src/components/Filter/Filter.module.scss:1]: I would prefer only having two columns/two categories per row. You can simply extend the height of the filter component to make it fit.

**Review by JosefineM2000 — DISMISSED** [review]

**Review by wernerramon — APPROVED** [review]
